javascript/arrays.js

```diff
@@ -0,0 +1,41 @@
+export default () =>{
+    let colors = ['red', 'orange', 'yellow'];
+    for (let color of colors){
+        console.log(color);
+    }
+    colors.forEach(el =>{
+        console.log(el);
+    });
+    function rotate(steps, arr){
+        for (let i = 0; i < steps; i++){
+            arr = [arr[arr.length - 1]].concat(arr.splice(0, arr.length-1));
+        }
+        return arr;
```

Comment:
nisam najbolje skontao zadatak ali ako je ideja bila da 
```
arr1 = [1,2,3,4,5]

steps = 3

roatateArr1 = [3, 4, 5, 1, 2]

```

onda bi to moglo ovako 
```
return [ ...arr.slice(arr.length - (steps % arr.length)) , ...arr.slice( 0, arr.length - (steps % arr.length))]
```